### PR TITLE
feat: change ui element titles and allow multiple

### PR DESCRIPTION
**Pull request by AntBush**
Implements # 1370

### CODE REVIEW

**Review by marcellmueller — COMMENTED** [review]
I noticed that the upload button jumps to the other side when you upload with this change. What do you think about enabling this with a `ui:option` flag then setting `flex-direction: column-reverse` maybe with a bit of a padding under the button?

**Comment by AntBush** [comment]
> I noticed that the upload button jumps to the other side when you upload with this change. What do you think about enabling this with a `ui:option` flag then setting `flex-direction: column-reverse` maybe with a bit of a margin under the button?

Yeah I noticed in the happo screens. I added the `ui:option` flag, but I didn't put a margin under the button, I'll go ahead and do that!

File: app/lib/theme/components/FileComponent.tsx [hunk 1]
```diff
@@ -184,6 +194,7 @@ const FileComponent: React.FC<FileComponentProps> = ({
       </StyledDetails>
       <div>
         <StyledButton
+          addBotttomMargin={wrap}
```
Comment (marcellmueller) [inline app/lib/theme/components/FileComponent.tsx:197]:
```suggestion
          addBottomMargin={wrap}
```

**Review by marcellmueller — APPROVED** [review]
Thanks, nice work! I noticed 1 minor issue that caused the margin to not work though other than that this is great!

File: app/lib/theme/components/FileComponent.tsx (hunk 1 above)
Comment (AntBush) [reply app/lib/theme/components/FileComponent.tsx]: <s>The extra t is for Tony</s> Fixed!

**Pull request merged**